### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by aabayomi**
missing __init__.py and minor changes in conversion.py

### CODE REVIEW

**Comment by ijnek** [comment]
Thanks @aabayomi for your contribution, the __init__ -> __init__.py change looks fine.
What's the reason for having to change the type hint from ``list[Marker]`` to ``Marker``  ?

**Comment by aabayomi** [comment]
> Thanks @aabayomi for your contribution, the **init** -> **init**.py change looks fine. What's the reason for having to change the type hint from `list[Marker]` to `Marker` ?

It's because there is a "TypeError: 'type' object is not subscriptable" in Marker (this seems to return an int32) MarkerArrray is an Array type. Not sure if you mean to use  MarkerArrray.

File: soccer_vision_3d_rviz_markers/conversion.py [hunk 1]
```diff
@@ -104,7 +104,7 @@ def obstacle_to_marker(msg: Obstacle) -> Marker:
     return marker
 
 
-def robot_to_markers(msg: Robot) -> list[Marker]:
+def robot_to_markers(msg: Robot) -> Marker:
```
Comment (Flova) [inline soccer_vision_3d_rviz_markers/conversion.py]: This function returns a list of Markers. But the previous notation was wrong as `list` should be capitalized and imported. 

Add this import: `from typing import List`
And change the type to `List[Marker]`

**Comment by aabayomi** [comment]
Oh got it. I have made the change.

File: soccer_vision_3d_rviz_markers/conversion.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import List
```
Comment (Flova) [inline soccer_vision_3d_rviz_markers/conversion.py]: List needs to be imported from `typing`, but I just noticed that from python 3.9 onwards list, dict, etc. are generic types out of the box so the original notation is correct.

See this pep: https://peps.python.org/pep-0585/

Comment (aabayomi) [reply soccer_vision_3d_rviz_markers/conversion.py]: Yes. I did in this commit 24e9e4a38c4113a88b0c580511b91b019ffc6093

Comment (Flova) [reply soccer_vision_3d_rviz_markers/conversion.py]: Ah nvm, I reviewed the non up to date version.

**Review by Flova — APPROVED** [review]

**Comment by ijnek** [comment]
Thanks for your contribution @aabayomi!

**Comment by aabayomi** [comment]
> Thanks for your contribution @aabayomi!

You are welcome. :)

**Pull request merged**